cli/tests/common/mod.rs

```diff
@@ -0,0 +1,205 @@
+use std::ffi::OsStr;
+use std::path::{Path, PathBuf};
+use std::{env, fs};
+
+pub use assert_cmd::assert::Assert;
+pub use assert_cmd::Command;
+use phylum_cli::api::{PhylumApi, PhylumApiError, ResponseError};
+use phylum_cli::config::{AuthInfo, Config, ConnectionInfo};
+use phylum_types::types::auth::RefreshToken;
+pub use predicates::prelude::*;
+use reqwest::StatusCode;
+use tempfile::TempDir;
+
+const HEADER: &str = "import { PhylumApi } from 'phylum';";
```

Comment:
Being a bit more descriptive on the name here is probably a good idea.

Just a really rough idea:

```suggestion
const TS_PHYLUM_IMPORT: &str = "import { PhylumApi } from 'phylum';";
```

Reply:
I was thinking to move this in `TestExtension::new`, maybe get rid of the constant altogether and hardcode it in the call to `fs::write` as it's not being used anywhere else. What do you think?

Reply:
I don't see a problem with that.